meerkat_frontend/src/files/pages/tabletCompletness.html

```diff
@@ -0,0 +1,81 @@
+<div class="top-bar row">
+
+	<div class="location-box box">
```

Comment:
Indentation in this file is done with tab characters instead of hard spaces?  Let's keep converting indentation to 4 hard spaces.